Makefile

```diff
@@ -105,7 +107,7 @@ test-helps:
 	$(foreach subcommand,$(SUBCOMMANDS), $(PYTHON) scripts/compose.py $(subcommand) --help >/dev/null || exit 1;)
 
 test-all: venv test-compose lint test-helps ## Run all the tests
-	pytest -v -s $(JUNIT_OPT)/all-junit.xml
+	pytest -v -s $(PYTEST_ARGS) $(JUNIT_OPT)/all-junit.xml
```

Comment:
IIRC if you have a `pytest.ini` and pass parameters in the CLI ignore the parameters in the `pytest.ini`

Reply:
I think that's correct. This is strictly for debugging in a few rare cases, such as cases where I wanted to pass `--pdb` in, for example. It shouldn't typically be used.